pkg/server/server_test.go

```diff
@@ -1563,3 +1563,60 @@ func TestDelegateCheckResolver(t *testing.T) {
 		require.True(t, ok)
 	})
 }
+
+func TestWriteAuthorizationModelWithExperimentalEnableModularModels(t *testing.T) {
+	ctx := context.Background()
+	storeID := ulid.Make().String()
+
+	mockController := gomock.NewController(t)
+	defer mockController.Finish()
+
+	mockDatastore := mockstorage.NewMockOpenFGADatastore(mockController)
+
+	s := MustNewServerWithOpts(
```

Comment:
nitpick: will it be better to move this inside the test "rejects_request_with_schema_version_1.2" to avoid shadow variable in the test "accepts_request_with_schema_version_1.2_if_experimental_flag_enabled"?

Reply:
Given the mock calls on `mockDatastore` do you think it makes sense to also move that into each test?

Reply:
Moved in e7679cdab99fce275ceedfaf1a137458b08de86a